babel.cfg

```diff
@@ -1,4 +1,4 @@
-[python: */**.py]
+[python: **.py]
```

Comment:
Thje original was correct if you run the babbel command from the root project folder